k8s.gcr.io/images/k8s-staging-etcd/images.yaml

```diff
@@ -1,6 +1,5 @@
 - name: etcd
   dmap:
-    "sha256:80d539b791923b37fb501166729307c987a237bf9a72cef3dbfca24744b5cb76": ["3.4.7-0", "3.4.7"]
```

Comment:
Removed based on @listx's [review](https://github.com/kubernetes/k8s.io/pull/889#issuecomment-632411745) to not overlap with the etcd backfill promotions in https://github.com/kubernetes/k8s.io/blob/master/k8s.gcr.io/images/DO-NOT-MODIFY-legacy-backfill/images.yaml.